### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by qjack001**
Some changes I think work better, plus quality of life stuff.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+	bundle = false
+	minify = true
 [build.processing.js]
-  bundle = false
-  minify = true
+	bundle = false
+	minify = true
 [build.processing.html]
-  pretty_urls = true
+	pretty_urls = false
```
Comment (qjack001) [inline netlify.toml:28]: Astro already does pretty URLs, I find having this setting on ends up with some strange results (going to `/about` redirects to `/about/`)

File: package.json [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-    "build": "astro build",
-    "preview": "astro preview"
-  },
-  "devDependencies": {
-    "@astrojs/renderer-vue": "^0.2",
-    "astro": "^0.22.20",
-    "autoprefixer": "^10.4.2",
-    "eslint-config-goodbyte-styleguide": "^0.0.7"
-  }
+	"name": "@goodbyteco/vivic",
```
Comment (qjack001) [inline package.json:2]: Not that important, but the rename gets rid of the IDE warning.

File: package.json [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
+	"version": "1.0.0",
+	"private": true,
+	"scripts": {
+		"dev": "astro dev",
+		"build": "astro build",
+		"preview": "astro preview",
+		"lint": "eslint ."
+	},
+	"dependencies": {
+		"@astrojs/renderer-vue": "^0.4.0"
```
Comment (qjack001) [inline package.json:12]: I know this is sort of weird. But we're loading Vue client-side for the `Work.vue` component. I'm honestly not sure, but I would assume this is the dep that's providing the Vue that gets sent.

Comment (qjack001) [reply package.json:12]: (also bumped from `0.2.0` -> `0.4.0`)

File: src/utils/stringUtils.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-      function(txt) {
-        return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
-      }
-    );
-  }
\ No newline at end of file
+	return str.replace(
+		/\w\S*/g,
+		function(txt) {
+			return txt.charAt(0).toUpperCase() + txt.substring(1).toLowerCase();
```
Comment (qjack001) [inline src/utils/stringUtils.ts:5]: `substr()` is deprecated. `substring()` does that same thing.

**Comment by qjack001** [comment]
I can't check the Netlify logs to see what's failing, but perhaps the bump to Astro 0.23.2 was too far.

**Comment by HoloPollock** [comment]
this issue is that node 12 is not supported did you delete the .nodeversion

**Review by HoloPollock — APPROVED** [review]

**Pull request merged**